### PR TITLE
Better hint for missed default network: add supporting for image export/instance import

### PR DESCRIPTION
**Pull request by dntczdx**
Supported for image import in the previous PR. Now add supporting for image export/instance import

### CODE REVIEW

File: cli_tools/common/utils/daisy/daisy_utils.go [hunk 1]
```diff
@@ -167,3 +167,15 @@ func RemovePrivacyLogTag(message string) string {
...
 }
+
+// PostProcessDErrorForNetworkFlag processes DError for network flag
+func PostProcessDErrorForNetworkFlag(action string, err error, network string, w *daisy.Workflow) {
+	if derr, ok := err.(daisy.DError); ok {
+		if derr.CausedByErrType("networkResourceDoesNotExist") && network == "" {
+			w.LogWorkflowInfo("A VPC network is required for running %v," +
+																" and the default VPC network does not exist in your project. You will need to" +
+																" specify a VPC network with the --network flag. For more information about" +
+																" VPC networks, see https://cloud.google.com/vpc.", action)
```
Comment (zoran15) [inline cli_tools/common/utils/daisy/daisy_utils.go]: blank lines

File: cli_tools/common/utils/daisy/daisy_utils.go [hunk 2]
```diff
@@ -167,3 +167,15 @@ func RemovePrivacyLogTag(message string) string {
 
 	return message
 }
+
+// PostProcessDErrorForNetworkFlag processes DError for network flag
```
Comment (zoran15) [inline cli_tools/common/utils/daisy/daisy_utils.go]: Add a better explanation of what "process" means here.

File: cli_tools/common/utils/daisy/daisy_utils.go [hunk 3]
```diff
@@ -167,3 +167,15 @@ func RemovePrivacyLogTag(message string) string {
...
+func PostProcessDErrorForNetworkFlag(action string, err error, network string, w *daisy.Workflow) {
+	if derr, ok := err.(daisy.DError); ok {
+		if derr.CausedByErrType("networkResourceDoesNotExist") && network == "" {
+			w.LogWorkflowInfo("A VPC network is required for running %v," +
```
Comment (zoran15) [inline cli_tools/common/utils/daisy/daisy_utils.go]: Does this even build? where the closing bracket for the func call?

File: cli_tools/gce_vm_image_import/importer/importer.go [hunk 4]
```diff
@@ -292,20 +292,9 @@ func Run(clientID string, imageName string, dataDisk bool, osID string, customTr
...
 	return w, nil
 }
-
-func postProcessErrorForNetwork(err error, network string, w *daisy.Workflow) {
-	if derr, ok := err.(daisy.DError); ok {
-		if derr.CausedByErrType("networkResourceDoesNotExist") && network == "" {
-			w.LogWorkflowInfo("A VPC network is required for running image import," +
-				" and the default VPC network does not exist in your project. You will need to" +
-				" specify a VPC network with the --network flag. For more information about" +
-				" VPC networks, see https://cloud.google.com/vpc.")
```
Comment (zoran15) [inline cli_tools/gce_vm_image_import/importer/importer.go]: Why is this long description gone from daisy_utils version of this func?

File: cli_tools/common/utils/daisy/daisy_utils.go (hunk 3 above)
Comment (dntczdx) [reply cli_tools/common/utils/daisy/daisy_utils.go]: It has no problem, I don't get...

File: cli_tools/gce_vm_image_import/importer/importer.go (hunk 4 above)
Comment (dntczdx) [reply cli_tools/gce_vm_image_import/importer/importer.go]: It's there. Would you double check?

File: cli_tools/common/utils/daisy/daisy_utils.go (hunk 3 above)
Comment (zoran15) [reply cli_tools/common/utils/daisy/daisy_utils.go]: Something was off with github showing this change, didn't see lines below.

File: cli_tools/gce_vm_image_import/importer/importer.go (hunk 4 above)
Comment (zoran15) [reply cli_tools/gce_vm_image_import/importer/importer.go]: Yep, something was off with github

**Pull request merged**